### ISSUE TITLE
[SIP] SharedStake Improvement Proposal Template

### ISSUE THREAD

**Issue by steao** [SharedStake/SharedStake-ui#18]
TL;DR: Should the SharedStake DAO formalize this template for future SIPs?

Summary: SIPs give every DAO member a voice in the direction of the protocol and should follow a template for efficiency. This SIP is an example of the proposed template style going forward until a new SIP proposes an updated format. The TL;DR section is the SIP's main proposal. The Summary will be 140 words or less to allow for single tweet dissemination. Since snapshot.page does not allow edits, a link should be created (ideally in a publicly accessible forum, social channel, etc.) before the SIP is on snapshot and the link added in the Discussion section on submission. The Note section will address anything required in the SIP that is not covered in prior sections. 

Discussion: [link to discussion]

Note: n/a

**Comment by sharedStake-main** [comment]
https://github.com/SharedStake/SIPs
